style/images/git-icon.svg

```diff
@@ -10,8 +10,8 @@
    version="1.1"
    inkscape:version="0.91 r13725"
    xml:space="preserve"
-   width="114.8625"
-   height="114.8625"
+   width="20"
+   height="20"
```

Comment:
If @telamonian think this should be an svg fix here rather than addressing this in lab (as I mentioned in my previous comment) then I believe this would be a more consistent fix with icon in lab
```suggestion
   width="20"
```

Reply:
Thanks makes sense to me, but maybe width="16" to match the other icons that were all changed to that width. Will wait to hear about the overall handling though.